### PR TITLE
Update GetPlayerFromServerId.md

### PR DESCRIPTION
**Pull request by AvarianKnight**

### CODE REVIEW

File: ext/native-decls/GetPlayerFromServerId.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Player GET_PLAYER_FROM_SERVER_ID(int serverId);
 
 
 ## Parameters
-* **serverId**: 
+* **serverId**: the players network id
```
Comment (blattersturm) [inline ext/native-decls/GetPlayerFromServerId.md:13]: the players (plural of player?) network id? hm?

File: ext/native-decls/GetPlayerFromServerId.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Player GET_PLAYER_FROM_SERVER_ID(int serverId);
...
 
 ## Return value
+The player index, or -1 if they didn't exist
```
Comment (blattersturm) [inline ext/native-decls/GetPlayerFromServerId.md:16]: i also end my sentences without anything

**Review by blattersturm — CHANGES_REQUESTED** [review]

File: ext/native-decls/GetPlayerFromServerId.md (hunk 1 above)
Comment (AvarianKnight) [reply ext/native-decls/GetPlayerFromServerId.md:13]: Yea shouldn't be plural 😆 Regarding network id you refer to this as the NetID in other parts of the scripting manual [like here](https://docs.fivem.net/docs/scripting-manual/networking/ids/), would you like me to just leave it as `the player's server id`?

File: ext/native-decls/GetPlayerFromServerId.md (hunk 2 above)
Comment (AvarianKnight) [reply ext/native-decls/GetPlayerFromServerId.md:16]: periods are for wimps 🤣 i'' fix it here in a few

**Comment by blattersturm** [comment]
Abandoned PR, will be taken in with the big native review.

**Pull request closed without merging**